test/ttmlir/Conversion/StableHLOToTTIR/dot_general/dot_general_2d.mlir

```diff
@@ -1,10 +1,11 @@
 // REQUIRES: stablehlo
 // RUN: ttmlir-opt --stablehlo-to-ttir-pipeline %s | FileCheck %s
 module @jit_dot_general attributes {} {
-  func.func public @test_dot_general(%arg0 : tensor<16x32xf32>, %arg1 : tensor<32x8xf32>) -> tensor<16x8xf32> {
-    %0 = stablehlo.dot_general %arg0, %arg1, contracting_dims = [1] x [0] : (tensor<16x32xf32>, tensor<32x8xf32>) -> tensor<16x8xf32>
-    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
-    // CHECK: %[[C:.*]] = "ttir.matmul"[[C:.*]]
+  func.func public @test_dot_general(%arg0 : tensor<16x32xf32>, %arg1 : tensor<8x32xf32>) -> tensor<16x8xf32> {
+    %0 = stablehlo.dot_general %arg0, %arg1, contracting_dims = [1] x [1] : (tensor<16x32xf32>, tensor<8x32xf32>) -> tensor<16x8xf32>
+    // CHECK: "ttir.dot_general"
+    // CHECK-SAME: <{batchdims_a = array<i64>, batchdims_b = array<i64>, contractdims_a = array<i64: 1>, contractdims_b = array<i64: 1>}
+    // CHECK-SAME: > : (tensor<16x32xf32>, tensor<8x32xf32>) -> tensor<16x8xf32>
```

Comment:
At least remove `>` from the beginning. I would also remove `:`, this can stay `(tensor<16x32xf32>, tensor<8x32xf32>) -> tensor<16x8xf32>`.